build/test/ProjectsToTest.props

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Layout" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+      <ItemGroup>
+            <ProjectsToTest Include="ArgumentForwardingTests"/>
+            <ProjectsToTest Include="crossgen.Tests"/>
```

Comment:
nit: can you use 2 spaces?  We should all align so when someone else tries to edit this, the editor doesn't try reformatting the whole document.